src/Swift.Bindings/src/Model/TypeDecl/ProtocolConformance.cs

```diff
@@ -7,9 +7,19 @@ namespace BindingsGeneration;
 /// <summary>
 /// Represents a protocol conformance.
 /// </summary>
-/// <param name="TargetType">The type spec of the target type of the conformance</param>
-/// <param name="ProtocolSpec">The protocol spec of the conformance</param>
-public record ProtocolConformance(
-    NamedTypeSpec TargetType,
-    NamedTypeSpec ProtocolSpec
+/// <param name="ConformingType">The type that conforms to the protocol</param>
+/// <param name="Protocol">The protocol that the type conforms to</param>
+public record TypeConformance(
+    SwiftTypeName ConformingType,
+    SwiftTypeName Protocol
+);
+
+/// <summary>
+/// Represents a generic parameter conformance.
+/// </summary>
+/// <param name="GenericParameter">The generic parameter</param>
+/// <param name="ProtocolSpec">The protocol that the generic parameter conforms to</param>
+public record GenericParameterConformance(
+    NamedTypeSpec GenericParameter,
+    SwiftTypeName ProtocolSpec
```

Comment:
Why is it called `ProtocolSpec` here, but in the `ProtocolConformance` it's called just `Protocol` - maybe I just don't know the difference.

Reply:
No reason, thanks for catching that!